src/pywrangler/wranglers.py

```diff
@@ -2,12 +2,14 @@
 and corresponding descriptions.
 
 """
-
+from typing import Any
 
 from pywrangler.base import BaseWrangler
 from pywrangler.util import sanitizer
 from pywrangler.util.types import TYPE_ASCENDING, TYPE_COLUMNS
 
+NULL = object()
```

Comment:
Thinking about the NULL value, a true NULL value (e.g. in spark) could be a valid marker value, too. Therefore, the name should not be NULL but rather `VALUENOTSET` which should be more concise. Nevertheless, values like `None` are `NULL` are edge case. Anyway, I believe it's worth allowing those values because it is unclear how future computation engines will treat any of these values.